### ISSUE TITLE
[Question] Can GitVersionTask be configured to only update specified Assembly*Versions and not others

### ISSUE THREAD

**Issue by kbatman37** [GitTools/GitVersion#1319]
Looking at the documentation for GitVersion, it appears that there are options for only allowing GitVersion to set certain version while allowing others to stay under manual control. As an example, I would like to update the AssemblyFileVersion but not the AssemblyVersion. I have tried adding the following to my csproj file

```
<PropertyGroup>
    <UpdateAssemblyInfo>true</UpdateAssemblyInfo>
    <GenerateAssemblyFileVersionAttribute>true</GenerateAssemblyFileVersionAttribute>
    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
  </PropertyGroup>
```

and have updated my AssemblyInfo.cs file

```
[assembly: AssemblyVersion("1.0.0.0")]
//[assembly: AssemblyFileVersion("1.0.0.0")]
```

but I still end up with both versions being set to 1.0.0.0.

Is this type of configuration possible using GitVersionTask? I can not find any good examples or clear documentation showing this type of setup. but I do see references [here](https://gitversion.readthedocs.io/en/latest/usage/msbuild-task/#conditional-control-tasks) in the documentation that indicate something like this may be possible.

PS: sorry if this is the wrong place for this type of question. I will happily move it if you point me someplace more appropriate.

**Comment by kbatman37** [comment]
+ @asbjornu Can you point me in the right direction on this?

**Comment by asbjornu** [comment]
`GitVersionTask.UpdateAssemblyInfo` turns the entire assembly info patching feature on or off, not individual parts of it. It seems like the command line interface gives you more control over these things than the MSBuild task.

**Comment by kbatman37** [comment]
@asbjornu Thanks for pointing me in the correct direction. It appears that you are correct and the command line version is capable of this use case.

For anyone searching this in the future:
1. Create the `GitVersion.yml` file using `gitversion init`
1. Add the `assembly-versioning-scheme: None` (or option applicable for your situation)

**Comment by asbjornu** [comment]
I'm happy you figured out a solution. The feature discrepancy between the `.exe` and MSBuild task is unintended and something we want to get fixed (see #883).
